repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -139,9 +143,12 @@ def scan_events(path):
     """ Scan JSON file containing PES events """
     try:
         events = parse_file(path)
-    except (ValueError, KeyError) as error:
-        raise StopActorExecutionError(
-            message='Found invalid or outdated PES data.',
-            details={'details': error, 'hint': 'Consult docs to retrieve updated data.'})
+    except (ValueError, KeyError) as value_error:
```

Comment:
Sorry for being pick, but I think "value_error" is a wrong name for this value since it can be a key error also. What about "error" or "err"